cypress/integration/data-browser/graphs.spec.js

```diff
@@ -1,171 +1,134 @@
+    cy.url().then((url) => {
+      url
+    })
+  })
+})
+
+describe("Tests user interaction with tabs", () => {
+  it("Starts on Graph tab and then switches to filer tab", () => {
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+    cy.get(
+      '[href="/data-browser/graphs/quarterly/info/filers"] > .section'
+    ).click(0, 0, {
+      force: true,
+    })
+    cy.url().should(
+      "eq",
+      `${baseURLToVisit}/data-browser/graphs/quarterly/info/filers`
+    )
+  })
+
+  it("Starts on Graph tab and then switches to faq tab", () => {
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+    cy.get('[href="/data-browser/graphs/quarterly/info/faq"] > .section').click(
+      0,
+      0,
+      {
+        force: true,
+      }
+    )
+    cy.url().should(
+      "eq",
+      `${baseURLToVisit}/data-browser/graphs/quarterly/info/faq`
+    )
+  })
+})
+
+describe("Graph Specific tests", () => {
+  it("Visits graph page and checks that the url contains correct query parameters", () => {
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+    // Regex check graphName, periodLow, periodHigh and visibleSeries in the url
+    cy.url().then((url) => {
+      url.match(/^([a-z]{2,}-?)+$/)
+    })
+    cy.url().then((url) => {
+      url.match(/periodLow=^20\d{2}-Q\d/)
+    })
+    cy.url().then((url) => {
+      url.match(/periodHigh=^20\d{2}-Q\d/)
+    })
+    cy.url().then((url) => {
+      url.match(/visibleSeries^([a-zA-Z-\/]+(%20)?[a-zA-Z-]*,?)+$/)
+    })
+  })
+
+  it("Visit graph page then selects a graph from the dropdown menu", () => {
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+
+    cy.get(".react-select__graph__value-container").click(0, 0, {
+      force: true,
+    })
+    cy.get("#react-select-2-option-0-0").click(0, 0, { force: true })
+    cy.url().should("include", "ltv")
+    cy.url().then((url) => {
+      url.match(/^([a-z]{2,}-?)+$/)
+    })
+  })
+
+  it("Visits graphs page, change periods, checks that the url updates with selected periods, clicks 'Show All Quarters' button and checks url updated with default periods", () => {
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+
+    cy.url().then((url) => {
+      cy.get(".react-select__period_start__value-container").click(0, 0, {
+        force: true,
+      })
+      cy.get("#react-select-3-option-3").click(0, 0, { force: true })
+      // Check url doesn't include base graph url start quarter
+      !url.includes(url)
+
+      cy.get(".react-select__period_end__value-container").click(0, 0, {
+        force: true,
+      })
+      cy.get("#react-select-4-option-6").click(0, 0, { force: true })
+      // Check URL doesn't include base graph url end quarter
+      !url.includes(url)
+
+      cy.get(".reset").click(0, 0, { force: true })
+      // Checks url was reset to base graph url which includes start and end quarters
+      url.includes(url)
+    })
+  })
```

Comment:
These tests are not recognizing that `url` is the same for all of these `url.includes` checks. Additionally, you're always using the same `url` variable, expecting it no equal/not-equal itself. Instead you should be capturing the URL, make a drop-down selection, compare the before/after. 
 
We also need to use one of Cypress' test methods in order for Cypress recognize when a failure has occurred. 
  
 Something like `cy.wrap(url).should('eq', prevUrl)`
  
 https://docs.cypress.io/api/commands/should#Syntax